libs/shared/src/utils/__snapshots__/getPlatformConfig.spec.ts.snap

```diff
@@ -225,6 +226,10 @@ exports[`getPlatformConfig > gets config for platform myparcel 1`] = `
         "only_recipient",
         "signature",
       ],
+      "smallPackagePickupCountries": [
+        "NL",
+        "BE",
```

Comment:
Is get logisch (en werkt het goed) dat `pickupCountries` alleen `NL` bevat, terwijl hier zowel NL als BE mogelijk zijn? (sanity check)